mappings/net/minecraft/client/render/WorldRenderer.mapping

```diff
@@ -171,25 +238,62 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 		ARG 3 updated
 	METHOD m_rxbalurb hasSkyBlockingEffect (Lnet/minecraft/unmapped/C_dcmtgrvm;)Z
 		ARG 1 camera
+	METHOD m_rzgnhorc isSectionCompiled (Lnet/minecraft/unmapped/C_hynzadkk;)Z
+	METHOD m_tamcgvjw offsetFrustum (Lnet/minecraft/unmapped/C_jwzjqevg;)Lnet/minecraft/unmapped/C_jwzjqevg;
 	METHOD m_tbgywtxm reload ()V
 	METHOD m_tgxmzbib getParticlesFramebuffer ()Lnet/minecraft/unmapped/C_xxwgvelc;
+	METHOD m_uptmaubw addLateDebugPass (Lnet/minecraft/unmapped/C_obrvgeej;Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_vidgpspy;)V
+		ARG 1 frameGraph
+		ARG 2 camera
+		ARG 3 fog
+	METHOD m_uyzrbqdi rebuildSections (Lnet/minecraft/unmapped/C_dcmtgrvm;)V
 	METHOD m_uzhmpsuv scheduleBlockRender (III)V
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD m_uzvxuudh collectVisibleEntities (Lnet/minecraft/unmapped/C_dcmtgrvm;Lnet/minecraft/unmapped/C_jwzjqevg;Ljava/util/List;)Z
+		ARG 2 frustum
+		ARG 3 into
 	METHOD m_vnssyyxd setWorld (Lnet/minecraft/unmapped/C_ghdnlrrw;)V
 		ARG 1 world
 	METHOD m_vznbfahy applyFrustum (Lnet/minecraft/unmapped/C_jwzjqevg;)V
 		ARG 1 frustum
+	METHOD m_wcyzbxhv countRenderedSections ()I
+	METHOD m_whptjcei addMainPass (Lnet/minecraft/unmapped/C_obrvgeej;Lnet/minecraft/unmapped/C_dcmtgrvm;Lorg/joml/Matrix4f;Lorg/joml/Matrix4f;Lnet/minecraft/unmapped/C_vidgpspy;ZZLnet/minecraft/unmapped/C_lrgsdfwv;Lnet/minecraft/unmapped/C_eslcbfsq;)V
+		ARG 1 frameGraph
+		ARG 3 modelViewMatrix
+		ARG 4 projectionMatrix
+		ARG 5 fog
+		ARG 6 blockOutline
+		ARG 7 entityOutline
+	METHOD m_wlewmnef addCloudRenderPass (Lnet/minecraft/unmapped/C_obrvgeej;Lorg/joml/Matrix4f;Lorg/joml/Matrix4f;Lnet/minecraft/unmapped/C_qaetpqjq;Lnet/minecraft/unmapped/C_vgpupfxx;FIF)V
+		ARG 1 frameGraph
+		ARG 2 viewMatrix
+		ARG 3 projectionMatrix
+		ARG 4 mode
+		ARG 5 camera
+		ARG 6 time
+		ARG 7 color
+		ARG 8 cloudHeight
 	METHOD m_xuzbjwzg canDrawEntityOutlines ()Z
 	METHOD m_xzqucjte scheduleChunkRender (IIIZ)V
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
 		ARG 4 important
 	METHOD m_ydwujepn resetTransparencyShader ()V
+	METHOD m_yimwxysz scheduleTranslucentResort (Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_tizlwnnv;)V
+		ARG 1 pos
+		ARG 2 layer
 	METHOD m_ypwknjzq getCloudsFramebuffer ()Lnet/minecraft/unmapped/C_xxwgvelc;
-	METHOD m_znnjtopn getChunkBuilderDebugString ()Ljava/lang/String;
+	METHOD m_yyzjbtrm getChunkBuilder ()Lnet/minecraft/unmapped/C_sjynkhfr;
+	METHOD m_yzsrltvt renderDestroyProgress (Lnet/minecraft/unmapped/C_cnszsxvd;Lnet/minecraft/unmapped/C_dcmtgrvm;Lnet/minecraft/unmapped/C_igrgeffe$C_insrqxjm;)V
```

Comment:
```suggestion
	METHOD m_yzsrltvt renderBreakingProgress (Lnet/minecraft/unmapped/C_cnszsxvd;Lnet/minecraft/unmapped/C_dcmtgrvm;Lnet/minecraft/unmapped/C_igrgeffe$C_insrqxjm;)V
```
unless there's a reason otherwise